modules/claims_api/app/controllers/claims_api/v0/claims_controller.rb

```diff
@@ -27,8 +27,7 @@ def show
                  serializer: ClaimsApi::AutoEstablishedClaimSerializer
         else
           begin
-            evss_claim_id = ClaimsApi::AutoEstablishedClaim.evss_id_by_token(params[:id]) || params[:id]
-            fetch_and_render_evss_claim(evss_claim_id)
+            fetch_and_render_evss_claim(fetch_or_error_local_claim_id || params[:id])
           rescue EVSS::ErrorMiddleware::EVSSError
             render json: { errors: [{ detail: 'Claim not found' }] },
                    status: :not_found
```

Comment:
I realize this isn't a change in this PR but this seems to be catching *any* EVSS error which doesn't necessarily mean Claim isn't found.

Wonder if the `fetch_or_error_local_claim_id` should rescue from `ActiveRecord::RecordNotFound` with a bang-ified `find_by!(id: params[:id])`

Reply:
That's a good idea, I'll add a ticket to address this...